### ISSUE TITLE
Logging does not have proper permissions on SLE Micro 5.1

### ISSUE THREAD

**Issue by rancher-max** [rancher/rancher#36364]
**Rancher Server Setup**
- Rancher version: v2.6.3
- Installation option (Docker install/Helm Chart):
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): helm, k3s v1.21.9+k3s1
- Proxy/Cert Details: 

**Information about the Cluster**
- Kubernetes version: 
v1.21.9+k3s1
- Cluster Type (Local/Downstream): 
Local

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom): 
Admin

   
**Describe the bug**

**_There are effectively two issues here that are probably related to the same thing. Providing details for both below._**
1. The `rancher-logging-k3s-journald-aggregator` daemonset's pods are stuck in a CrashLoopBackoff. Getting the logs gives:
```
$ k logs -n cattle-logging-system pod/rancher-logging-k3s-journald-aggregator-jkcnh -f
Fluent Bit v1.8.8
* Copyright (C) 2019-2021 The Fluent Bit Authors
* Copyright (C) 2015-2018 Treasure Data
* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
* https://fluentbit.io/

[2022/02/02 20:54:14] [ info] [engine] started (pid=1)
[2022/02/02 20:54:14] [ info] [storage] version=1.1.4, initializing...
[2022/02/02 20:54:14] [ info] [storage] in-memory
[2022/02/02 20:54:14] [ info] [storage] normal synchronization mode, checksum disabled, max_chunks_up=128
[2022/02/02 20:54:14] [ info] [cmetrics] version=0.2.2
[2022/02/02 20:54:14] [error] [input:systemd:systemd.0] could not open the Journal
[2022/02/02 20:54:14] [error] [input:systemd:systemd.0] cannot initialize
[2022/02/02 20:54:14] [error] Failed initialize input systemd.0
[2022/02/02 20:54:14] [error] [lib] backend failed
```
I tried both `/var/log/journal` and `/run/log/journal` with the same result on both. I believe `/var/log/journal` is correct though based on https://github.com/rancher/charts/blob/dev-v2.6/charts/rancher-logging/100.0.1%2Bup3.15.0/app-readme.md#systemd-configuration; when I run the command mentioned there, I receive `auto` and I do have `/var/log/journal` on the node.

2. After setting up a ClusterOutput and ClusterFlow, then navigating to the logs for `rancher-logging-fluentbit`, I see a constant stream of the same error. Taking snippet from the beginning of the logs:
```
k logs -n cattle-logging-system pod/rancher-logging-fluentbit-2966s
Fluent Bit v1.8.8
* Copyright (C) 2019-2021 The Fluent Bit Authors
* Copyright (C) 2015-2018 Treasure Data
* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
* https://fluentbit.io

[2022/02/02 20:53:00] [ info] [engine] started (pid=1)
[2022/02/02 20:53:00] [ info] [storage] version=1.1.4, initializing...
[2022/02/02 20:53:00] [ info] [storage] root path '/buffers'
[2022/02/02 20:53:00] [ info] [storage] normal synchronization mode, checksum disabled, max_chunks_up=128
[2022/02/02 20:53:00] [ info] [storage] backlog input plugin: storage_backlog.1
[2022/02/02 20:53:00] [ info] [cmetrics] version=0.2.2
[2022/02/02 20:53:00] [error] [input:tail:tail.0] read error, check permissions: /var/log/containers/*.log
[2022/02/02 20:53:00] [ warn] [input:tail:tail.0] error scanning path: /var/log/containers/*.log
[2022/02/02 20:53:00] [ info] [input:storage_backlog:storage_backlog.1] queue memory limit: 95.4M
[2022/02/02 20:53:00] [ info] [filter:kubernetes:kubernetes.0] https=1 host=kubernetes.default.svc port=443
[2022/02/02 20:53:00] [ info] [filter:kubernetes:kubernetes.0] local POD info OK
[2022/02/02 20:53:00] [ info] [filter:kubernetes:kubernetes.0] testing connectivity with API server...
[2022/02/02 20:53:00] [ info] [filter:kubernetes:kubernetes.0] connectivity OK
[2022/02/02 20:53:00] [ info] [sp] stream processor started
[2022/02/02 20:53:04] [error] [input:tail:tail.0] read error, check permissions: /var/log/containers/*.log
[2022/02/02 20:53:04] [ warn] [input:tail:tail.0] error scanning path: /var/log/containers/*.log
[2022/02/02 20:53:09] [error] [input:tail:tail.0] read error, check permissions: /var/log/containers/*.log
[2022/02/02 20:53:09] [ warn] [input:tail:tail.0] error scanning path: /var/log/containers/*.log
[2022/02/02 20:53:14] [error] [input:tail:tail.0] read error, check permissions: /var/log/containers/*.log
[2022/02/02 20:53:14] [ warn] [input:tail:tail.0] error scanning path: /var/log/containers/*.log
```
Debugging the `/var/log/containers/` location produced the following results:
- Output of ls with selinux context included:
```
$ ls -ltraZ /var/log/containers/
total 204
drwxr-xr-x. 1 root root system_u:object_r:var_log_t:s0         664 Feb  2 18:28 ..
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   131 Feb  2 18:28 local-path-provisioner-84bb864455-ljchk_kube-system_local-path-provisioner-515c9ed91c7d948cd1aeff87514f1d51e8436ec610c05288f95b952051b302ea.log -> /var/log/pods/kube-system_local-path-provisioner-84bb864455-ljchk_dc66c7dd-c327-4228-8327-3813a018f8ec/local-path-provisioner/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   101 Feb  2 18:28 coredns-6c46d74d64-h7w66_kube-system_coredns-0b454099935c834dd34e77957e9e71a4198804b5ad77caa3ad933222f082181e.log -> /var/log/pods/kube-system_coredns-6c46d74d64-h7w66_5cc8a2dc-68dd-4a79-acf1-0d7881816719/coredns/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   114 Feb  2 18:28 metrics-server-ff9dbcb6c-dffn9_kube-system_metrics-server-ecbe7aa4c25a5b1427379efa56e80f3b538345f2bfdd5bb2ac3bbefa888ab151.log -> /var/log/pods/kube-system_metrics-server-ff9dbcb6c-dffn9_ede4924b-9c8b-49c2-9717-3fe35caadd67/metrics-server/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   104 Feb  2 18:28 helm-install-traefik-crd-m8v4g_kube-system_helm-44f9eac1cd4db55c1ed9b085ba3caf6fd8b0038edeffe1ebff65f072688a145b.log -> /var/log/pods/kube-system_helm-install-traefik-crd-m8v4g_9e9a11f8-8403-48b3-9d50-f8bc14417775/helm/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   100 Feb  2 18:29 helm-install-traefik-c6n5z_kube-system_helm-657eceeca5d9203df903b303be54cbc7e33b81f3a911a26dc52cd365482720f3.log -> /var/log/pods/kube-system_helm-install-traefik-c6n5z_73ef5f8c-adad-4476-a667-db0303ffc3ee/helm/2.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0    99 Feb  2 18:29 svclb-traefik-56lln_kube-system_lb-port-80-ca7a75b891fb3e654f4d4ce96bde8674e777a47050a19b2c83d95eb28d382b0a.log -> /var/log/pods/kube-system_svclb-traefik-56lln_94fe04a6-0aa0-4cce-be9f-9ba7ac6e347c/lb-port-80/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   100 Feb  2 18:29 svclb-traefik-56lln_kube-system_lb-port-443-68edfdbd3d9930bd95c67adfef49e865c45ca7ee3adb9d38d93c32fb5fe16fb5.log -> /var/log/pods/kube-system_svclb-traefik-56lln_94fe04a6-0aa0-4cce-be9f-9ba7ac6e347c/lb-port-443/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   101 Feb  2 18:29 traefik-55fdc6d984-2zvtk_kube-system_traefik-92fb3424ef5cd7f25ba2e1ef27b14169e9c06515757584218d03f33b46ef5645.log -> /var/log/pods/kube-system_traefik-55fdc6d984-2zvtk_4ecd726a-bc65-45a7-869a-748039d56a6d/traefik/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   120 Feb  2 18:33 cert-manager-webhook-5fd7d458f7-lm47h_cert-manager_cert-manager-da5fc1b441b3ba38a6149980246775a3c7b53684a2d756dd4ab74d0ce72be3dd.log -> /var/log/pods/cert-manager_cert-manager-webhook-5fd7d458f7-lm47h_2f4d45de-3064-493e-af92-bedab9897193/cert-manager/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   123 Feb  2 18:33 cert-manager-cainjector-856d4df858-fmp6h_cert-manager_cert-manager-3f5cecbe6921db0a0b3772a615726ce05de3a5136452cc236a56bec79393750c.log -> /var/log/pods/cert-manager_cert-manager-cainjector-856d4df858-fmp6h_bfe117ab-e72a-49b0-910b-8ebad0d19198/cert-manager/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   112 Feb  2 18:33 cert-manager-66b6d6bf59-m58tb_cert-manager_cert-manager-f7e680e49165ae8a5bf34e491e06fc498a44bf1b6e71bf40526a6c7df8c01459.log -> /var/log/pods/cert-manager_cert-manager-66b6d6bf59-m58tb_4a344df4-73f1-43a9-b387-8b69b578c138/cert-manager/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   103 Feb  2 18:45 rancher-6bcbdd6cb7-z77rp_cattle-system_rancher-9f7dd534d964f298d8224eb93e74325d032eb23a703e5270de6eac3c394f0387.log -> /var/log/pods/cattle-system_rancher-6bcbdd6cb7-z77rp_86ec7b75-66ed-469b-b96e-4587f5f0f7e0/rancher/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   103 Feb  2 18:45 rancher-6bcbdd6cb7-8pszw_cattle-system_rancher-030f26da88c57e0073a570887e5c8ae6166c9ed55f66ab1fea46281cb858a26e.log -> /var/log/pods/cattle-system_rancher-6bcbdd6cb7-8pszw_6ac92c8b-8386-4f5b-aeab-24a73c53c1ab/rancher/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   103 Feb  2 18:45 rancher-6bcbdd6cb7-ctf7w_cattle-system_rancher-b9592fa17edbb4f998503bc9d00163805ff2630e0729239bb1df484966e91120.log -> /var/log/pods/cattle-system_rancher-6bcbdd6cb7-ctf7w_8a09d9f3-8631-4435-b29d-7b4a01434fdc/rancher/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   127 Feb  2 18:46 fleet-controller-6666887949-rxjtj_cattle-fleet-system_fleet-controller-d023b13c1b2106bd8e60148e1cb66d5dc723ce93978d7e755cf3830c3f52089b.log -> /var/log/pods/cattle-fleet-system_fleet-controller-6666887949-rxjtj_6bd5770e-d5ab-4505-b585-08d5c63e8f4b/fleet-controller/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   107 Feb  2 18:46 gitjob-7b97c9c7fd-5c5bk_cattle-fleet-system_gitjob-242c78ec3a592e4817077439519f2aeb7ad0ef47ecd5d83b6e178a6c2a4f73d4.log -> /var/log/pods/cattle-fleet-system_gitjob-7b97c9c7fd-5c5bk_19a28dd0-1100-483f-bb95-bb9bf959a86d/gitjob/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   119 Feb  2 18:47 rancher-webhook-5d4f5b7f6d-fzzm5_cattle-system_rancher-webhook-799723855c545e54f185ae3ac1199faf4aa558750ae581728c3aa6f626e07201.log -> /var/log/pods/cattle-system_rancher-webhook-5d4f5b7f6d-fzzm5_db034c27-2c26-4b04-9140-0739c28e1c65/rancher-webhook/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   123 Feb  2 18:47 fleet-agent-78f694664b-h8f2j_cattle-fleet-local-system_fleet-agent-c7392e8a7c50eb2519cbacf73c87c7e5ebcee18bd1034295bdc8849753921295.log -> /var/log/pods/cattle-fleet-local-system_fleet-agent-78f694664b-h8f2j_6de4d8a7-3b8f-4ce1-915f-e18042a804dd/fleet-agent/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   119 Feb  2 19:12 cis-operator-76698795fd-lhklm_cis-operator-system_cis-operator-dbaaed0afcd8711546c8c7b382ff1764ad9d4906d154e888d7eb5b74e98b7389.log -> /var/log/pods/cis-operator-system_cis-operator-76698795fd-lhklm_144d9c71-edd1-4de0-a2cb-684bb9413c67/cis-operator/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   127 Feb  2 20:52 rancher-logging-55fd5846c7-mxn2s_cattle-logging-system_rancher-logging-cf6382e531ab2e75836a59177998d1fc82a83ac654e59e21a009b33689a26dd9.log -> /var/log/pods/cattle-logging-system_rancher-logging-55fd5846c7-mxn2s_2d296b65-9965-43af-8c5e-bb037331cac6/rancher-logging/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   112 Feb  2 20:52 rancher-logging-fluentd-0_cattle-logging-system_fluentd-33216636e8ec6ac3fea1fbf24517f6bae2e023f3ce48543fc9b69207c57d3e05.log -> /var/log/pods/cattle-logging-system_rancher-logging-fluentd-0_a64c4549-677f-4d7a-b1d8-57e2de191b37/fluentd/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   121 Feb  2 20:53 rancher-logging-fluentbit-2966s_cattle-logging-system_fluent-bit-05e7cb07beaf2a5727b9cf31d3f395155d68e043b33a73f775f6a0353ac8bed4.log -> /var/log/pods/cattle-logging-system_rancher-logging-fluentbit-2966s_fdd239a6-ed81-4d53-8bc5-0ae884f36faf/fluent-bit/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   120 Feb  2 20:53 rancher-logging-fluentd-0_cattle-logging-system_config-reloader-f41c1c24514c6e9cc0c39b8b9fe5f9c20997ad9d1533ad2bd119d380170a9841.log -> /var/log/pods/cattle-logging-system_rancher-logging-fluentd-0_a64c4549-677f-4d7a-b1d8-57e2de191b37/config-reloader/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   128 Feb  2 21:02 rancher-alerting-drivers-prom2teams-76cdfbc6d4-8vc7z_default_prom2teams-7fffdfe68719248dce4e4f83652dad70c0dee020896088f835dbb904b9cead93.log -> /var/log/pods/default_rancher-alerting-drivers-prom2teams-76cdfbc6d4-8vc7z_d58f4d98-56b6-4417-9918-6f3fd0bbb515/prom2teams/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   130 Feb  2 21:05 pushprox-k3s-server-client-fdt6p_cattle-monitoring-system_pushprox-client-d6b0a7a6f10cdd1130411e60e759ae23dfbb2e2815d8803a0258e344d82c073e.log -> /var/log/pods/cattle-monitoring-system_pushprox-k3s-server-client-fdt6p_15201eb5-2916-43be-95a0-a657251e3f0b/pushprox-client/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   145 Feb  2 21:05 rancher-monitoring-prometheus-node-exporter-qcggd_cattle-monitoring-system_node-exporter-2f1cd812e6a9fe26e6b368cac2a0daff1e413caabe6a2952f627caaa726403be.log -> /var/log/pods/cattle-monitoring-system_rancher-monitoring-prometheus-node-exporter-qcggd_9fcb1eb4-3ebe-4843-ad89-c729c3bd9956/node-exporter/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   138 Feb  2 21:05 pushprox-k3s-server-proxy-f4f5d4874-rrj66_cattle-monitoring-system_pushprox-proxy-d623de5ace51c8622bfec2e3c683eab0988f966d3ef1c3df3c1df2081380c209.log -> /var/log/pods/cattle-monitoring-system_pushprox-k3s-server-proxy-f4f5d4874-rrj66_60a68896-b606-4849-a0a4-72b65f78e135/pushprox-proxy/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   145 Feb  2 21:05 rancher-monitoring-operator-559767d69b-bf27s_cattle-monitoring-system_rancher-monitoring-7bb3f915fba93beb501f77e4edfcf13a591edb96e7ac7b6ac91f8e090c0416a3.log -> /var/log/pods/cattle-monitoring-system_rancher-monitoring-operator-559767d69b-bf27s_a03bd1a6-c569-4623-b225-94885102eb90/rancher-monitoring/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   154 Feb  2 21:05 rancher-monitoring-prometheus-adapter-8846d4757-h2x52_cattle-monitoring-system_prometheus-adapter-dc9ac97f5e9bbad82aa17e9c4b0d65015d5968cf2da3590cad4a060cd9530805.log -> /var/log/pods/cattle-monitoring-system_rancher-monitoring-prometheus-adapter-8846d4757-h2x52_9a2f6f53-ea35-43f1-bcb9-3c443a1430ea/prometheus-adapter/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   155 Feb  2 21:05 rancher-monitoring-kube-state-metrics-5bc8bb48bd-jtrqj_cattle-monitoring-system_kube-state-metrics-52900bbff24ebb373bd6a8284dad747888dd820310fd605b73409e208ae3f548.log -> /var/log/pods/cattle-monitoring-system_rancher-monitoring-kube-state-metrics-5bc8bb48bd-jtrqj_c4d2eb38-757c-4959-97a2-d8455d04e38c/kube-state-metrics/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   148 Feb  2 21:05 rancher-monitoring-grafana-57777cc795-l5kcx_cattle-monitoring-system_grafana-sc-datasources-6139305f0f1785c574eb14a9ebf7146f79b1b217125537f8b33bdb01ff1eac55.log -> /var/log/pods/cattle-monitoring-system_rancher-monitoring-grafana-57777cc795-l5kcx_36824973-11e4-42e9-9901-953bdba95984/grafana-sc-datasources/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   146 Feb  2 21:05 rancher-monitoring-grafana-57777cc795-l5kcx_cattle-monitoring-system_grafana-sc-dashboard-be5a22d9765ddc6b2a668cd6698eff2c92ddac9493f1bdbc7aa3ca0b384d08fb.log -> /var/log/pods/cattle-monitoring-system_rancher-monitoring-grafana-57777cc795-l5kcx_36824973-11e4-42e9-9901-953bdba95984/grafana-sc-dashboard/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   141 Feb  2 21:05 alertmanager-rancher-monitoring-alertmanager-0_cattle-monitoring-system_alertmanager-5daa61e8992e0288002d809acccf50ed84261cd57b473c61e5a698db56b791a2.log -> /var/log/pods/cattle-monitoring-system_alertmanager-rancher-monitoring-alertmanager-0_1ecf85f2-7ab6-4310-b4f2-5df9744e9553/alertmanager/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   145 Feb  2 21:05 prometheus-rancher-monitoring-prometheus-0_cattle-monitoring-system_init-config-reloader-ba2849d06e4deec085ef03015c1218c40895bae54a0638c6df116b6a08345735.log -> /var/log/pods/cattle-monitoring-system_prometheus-rancher-monitoring-prometheus-0_7fa949c5-d6f1-4dfc-91bc-8c414153f4f3/init-config-reloader/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   144 Feb  2 21:05 alertmanager-rancher-monitoring-alertmanager-0_cattle-monitoring-system_config-reloader-4a944b207872c8bcb9ee876eee419e20defb60c1f86eb49331369ea6d8cc443d.log -> /var/log/pods/cattle-monitoring-system_alertmanager-rancher-monitoring-alertmanager-0_1ecf85f2-7ab6-4310-b4f2-5df9744e9553/config-reloader/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   133 Feb  2 21:05 rancher-monitoring-grafana-57777cc795-l5kcx_cattle-monitoring-system_grafana-f91bd9195e7f1b64387de66a76ebb212be0fbcccec2a2f0e8dbaab685b0aa0a2.log -> /var/log/pods/cattle-monitoring-system_rancher-monitoring-grafana-57777cc795-l5kcx_36824973-11e4-42e9-9901-953bdba95984/grafana/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   135 Feb  2 21:06 prometheus-rancher-monitoring-prometheus-0_cattle-monitoring-system_prometheus-d47d4acb38d8e3606e31dd836d1d83f7f37f0bf2f6ef649954ba7a0dc2490a70.log -> /var/log/pods/cattle-monitoring-system_prometheus-rancher-monitoring-prometheus-0_7fa949c5-d6f1-4dfc-91bc-8c414153f4f3/prometheus/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   140 Feb  2 21:06 prometheus-rancher-monitoring-prometheus-0_cattle-monitoring-system_config-reloader-cf54a00f23e3c373173a87cbf6caa3bd0801519d9fc62db3b6f23e424a410d9c.log -> /var/log/pods/cattle-monitoring-system_prometheus-rancher-monitoring-prometheus-0_7fa949c5-d6f1-4dfc-91bc-8c414153f4f3/config-reloader/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   141 Feb  2 21:06 prometheus-rancher-monitoring-prometheus-0_cattle-monitoring-system_prometheus-proxy-2e3aa41aaaf0b69c63d2585b91adabdafadf72c4547f8d0dd6ed902930e49edd.log -> /var/log/pods/cattle-monitoring-system_prometheus-rancher-monitoring-prometheus-0_7fa949c5-d6f1-4dfc-91bc-8c414153f4f3/prometheus-proxy/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   139 Feb  2 21:06 rancher-monitoring-grafana-57777cc795-l5kcx_cattle-monitoring-system_grafana-proxy-a5c0e3cf616a7eb4d8555887cb7da653fdbfe48342dd9976ab5ecf1108c865e9.log -> /var/log/pods/cattle-monitoring-system_rancher-monitoring-grafana-57777cc795-l5kcx_36824973-11e4-42e9-9901-953bdba95984/grafana-proxy/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0    98 Feb  2 21:11 kiali-7bbd7cffbf-8q2xv_istio-system_kiali-9425e9fda89fded5d662808816ea2c5f3a947e95f0bd9601b9eb27cee6c56946.log -> /var/log/pods/istio-system_kiali-7bbd7cffbf-8q2xv_487e6fed-2207-4bb3-b455-1917bf7f8f97/kiali/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   103 Feb  2 21:12 istiod-5b4d98d68d-wl6kz_istio-system_discovery-c37861aa94818f04b09b8729241169560947d34de4259e401899dc0313358ebd.log -> /var/log/pods/istio-system_istiod-5b4d98d68d-wl6kz_f06f42e0-46f3-414a-8047-2ad894772796/discovery/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   119 Feb  2 21:12 istio-ingressgateway-5659c66658-6r52p_istio-system_istio-proxy-79d1aa8c23cfe3cf627a923dd90de0bed396be457d4280d02ede71d164c44cf2.log -> /var/log/pods/istio-system_istio-ingressgateway-5659c66658-6r52p_bcd56a1f-0584-40eb-8dcb-152ae08c7694/istio-proxy/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   135 Feb  2 21:20 gatekeeper-controller-manager-6668f7c79-7gh6x_cattle-gatekeeper-system_manager-900175bca9de285b0c27ae159542f5568e818f2360d55477efc50b8a324f6487.log -> /var/log/pods/cattle-gatekeeper-system_gatekeeper-controller-manager-6668f7c79-7gh6x_c3bd4cc4-12d9-4c10-88a0-c1daba17ae11/manager/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   121 Feb  2 21:20 gatekeeper-audit-b6bf64d6-gzcs6_cattle-gatekeeper-system_manager-c73e53aa6bb3938043a53f6c595837a0a18934447ac507cda6cf233e07cdabdf.log -> /var/log/pods/cattle-gatekeeper-system_gatekeeper-audit-b6bf64d6-gzcs6_ac309454-3dff-45d0-8962-489496e17b30/manager/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   135 Feb  2 21:20 gatekeeper-controller-manager-6668f7c79-c7hhj_cattle-gatekeeper-system_manager-39a900e382139c85fa9436573428c66a1ada4a526d8eb8c0df4a579724d20b53.log -> /var/log/pods/cattle-gatekeeper-system_gatekeeper-controller-manager-6668f7c79-c7hhj_aaf4bd84-ebaf-4231-a3b3-65b351e2139a/manager/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   135 Feb  2 21:20 gatekeeper-controller-manager-6668f7c79-zx59s_cattle-gatekeeper-system_manager-7b15cb19eab37db1a66cec6fa1d8bec1141b92d542e355d1d46e2a82a3d877e0.log -> /var/log/pods/cattle-gatekeeper-system_gatekeeper-controller-manager-6668f7c79-zx59s_30c0a756-1e18-4d07-a798-f844c6a8ce16/manager/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0    99 Feb  2 21:24 nginx-6dcb77f6b8-qwrpb_default_container-0-429fcc9c372b247488101d4c4326c1dbf56e4a3aaa1f239564fde99e585082db.log -> /var/log/pods/default_nginx-6dcb77f6b8-qwrpb_ed62bb23-0f1d-4859-952c-ef13e5e136d7/container-0/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   127 Feb  2 21:25 rancher-backup-7dfff6b5dd-47gbd_cattle-resources-system_rancher-backup-e7fca9818c5a5add149c7c1f4be573c95e7bd978e4500bd88549f469d00335e0.log -> /var/log/pods/cattle-resources-system_rancher-backup-7dfff6b5dd-47gbd_b190be39-2fc6-4a5a-89b4-39e189e50741/rancher-backup/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   130 Feb  2 22:41 rancher-logging-fluentd-configcheck-e371cea_cattle-logging-system_fluentd-6d17d5a8fe559adba5fcf0843f642f61339e077c58e23ea7f43607e6c3094196.log -> /var/log/pods/cattle-logging-system_rancher-logging-fluentd-configcheck-e371cea_fbe747de-06bf-44cf-a2e5-b41921ee6875/fluentd/0.log
lrwxrwxrwx. 1 root root system_u:object_r:container_log_t:s0   135 Feb  2 22:57 rancher-logging-k3s-journald-aggregator-lvtgv_cattle-logging-system_fluentbit-319691f63d59344bb8b161b306552308a14085890ae86fa2e6e359e6b6f65ef1.log -> /var/log/pods/cattle-logging-system_rancher-logging-k3s-journald-aggregator-lvtgv_de2678aa-ff49-4133-9ef3-cf38b4883c89/fluentbit/28.log
drwxr-xr-x. 1 root root system_u:object_r:container_log_t:s0 13888 Feb  2 22:57 .
```
- Trying to `cat` a file there:
```
$ cat /var/log/containers/alertmanager-rancher-monitoring-alertmanager-0_cattle-monitoring-system_alertmanager-5daa61e8992e0288002d809acccf50ed84261cd57b473c61e5a698db56b791a2.log
cat: /var/log/containers/alertmanager-rancher-monitoring-alertmanager-0_cattle-monitoring-system_alertmanager-5daa61e8992e0288002d809acccf50ed84261cd57b473c61e5a698db56b791a2.log: Permission denied
```
- Using `sudo`:
```
$ sudo cat /var/log/containers/alertmanager-rancher-monitoring-alertmanager-0_cattle-monitoring-system_alertmanager-5daa61e8992e0288002d809acccf50ed84261cd57b473c61e5a698db56b791a2.log
2022-02-02T21:05:51.446786009Z stderr F level=info ts=2022-02-02T21:05:51.446Z caller=main.go:221 msg="Starting Alertmanager" version="(version=0.22.2, branch=HEAD, revision=44f8adc06af5101ad64bd8b9c8b18273f2922051)"
2022-02-02T21:05:51.446818526Z stderr F level=info ts=2022-02-02T21:05:51.446Z caller=main.go:222 build_context="(go=go1.16.4, user=root@b595c7f32520, date=20210602-07:50:37)"
2022-02-02T21:05:51.511240427Z stderr F level=info ts=2022-02-02T21:05:51.510Z caller=coordinator.go:113 component=configuration msg="Loading configuration file" file=/etc/alertmanager/config/alertmanager.yaml
2022-02-02T21:05:51.511289593Z stderr F level=info ts=2022-02-02T21:05:51.511Z caller=coordinator.go:126 component=configuration msg="Completed loading of configuration file" file=/etc/alertmanager/config/alertmanager.yaml
2022-02-02T21:05:51.518414299Z stderr F level=info ts=2022-02-02T21:05:51.516Z caller=main.go:514 msg=Listening address=:9093
2022-02-02T21:05:51.518434142Z stderr F level=info ts=2022-02-02T21:05:51.517Z caller=tls_config.go:191 msg="TLS is disabled." http2=false
2022-02-02T21:05:54.40051562Z stderr F level=info ts=2022-02-02T21:05:54.400Z caller=coordinator.go:113 component=configuration msg="Loading configuration file" file=/etc/alertmanager/config/alertmanager.yaml
2022-02-02T21:05:54.400998964Z stderr F level=info ts=2022-02-02T21:05:54.400Z caller=coordinator.go:126 component=configuration msg="Completed loading of configuration file" file=/etc/alertmanager/config/alertmanager.yaml
```

**To Reproduce**
 
1. This probably happens on downstream SLE Micro k3s clusters as well, but just create a cluster using k3s v1.21.9+k3s1 on SLE Micro 5.1 and either import it into rancher or run rancher on it
2. Install logging app with default values. _This will reproduce issue 1 above_
3. Add a ClusterOutput and ClusterFlow
4. View logs for `rancher-logging-fluentbit`. _This will reproduce issue 2 above_

**Result**
See issues 1 and 2 in the Description above.

**Expected Result**

There should be no errors upon initially installing logging. When adding flows, it should work correctly and be able to retrieve log files.

**Screenshots**

N/A

**Additional context**

N/A

**Comment by superseb** [comment]
@rancher-max Can you share if any prereqs were applied as stated on https://rancher.com/docs/rancher/v2.6/en/security/selinux/? It seems we are waiting for support on this in https://github.com/rancher/rancher-selinux/pull/10, then rancher-logging needs to be configured as described on https://rancher.com/docs/rancher/v2.6/en/security/selinux/#configuring-the-logging-application-to-work-with-selinux.

Did it work when you disabled SELinux and/or did you check `/var/log/audit/audit.log` for denied log lines?

**Comment by rancher-max** [comment]
Ah this was my first time doing logging v2 on an selinux system, so I probably missed something there. I will try all of those things and get back to you.

The selinux rpms should've been applied during k3s install, after which I performed a reboot as required by transactional-update, although this does _not_ apply the `rancher-selinux` rpm. 
```
# See this snippet from https://get.k3s.io
    ${package_installer} install -y container-selinux
    ${package_installer} install -y https://${rpm_site}/k3s/${rpm_channel}/common/${rpm_site_infix}/noarch/k3s-selinux-0.4-1.${rpm_target}.noarch.rpm
```

Also, k3s does not enable audit logs by default, so I will do the following in my next attempt and post the results here:
1. Install the `rancher-selinux` RPM before performing the k3s install
2. Enable audit logs during k3s install
3. Ensure that I set `global.seLinux.enabled=true` in values.yaml when installing logging

If this is successful, I will start paring down what critically needs to be set from there. Thanks Seb!

**Comment by rancher-max** [comment]
@superseb Enabling `global.seLinux.enabled=true` in values.yaml when installing logging seemed to do the trick and resolved both issues mentioned in this issue.

I will note that trying to install the `rancher-selinux` rpm seemed to fail. I followed the general steps from https://rancher.com/docs/rancher/v2.6/en/security/selinux/#configuring-the-logging-application-to-work-with-selinux, but there isn't an rpm built yet for SLE Micro at this time. See https://github.com/rancher/rancher-selinux. I'm not sure what other ramifications this might have, but I would assume that not having the RPM installed properly could cause other issues with running Rancher successfully on this cluster. 

Because of this, I think this issue can be closed. It's likely we should open another issue to create a rancher-selinux rpm for sle micro (similar to what was done [here](https://github.com/k3s-io/k3s-selinux)), but I will defer to you on all of that.

**Comment by superseb** [comment]
I think we need to wait on https://github.com/rancher/rancher-selinux/pull/10 and re-test

**Comment by ronhorton** [comment]
**Pass** Verified in `2.6-head Commit ID eb03517`

successfully spun up a cluster with the new sle linux micro ami (ami-0d3c317f604c2c84c)

1. imported k3s cluster
2. created syslog server
3. created clusterflow and clusteroutput
4. logs appeared in syslog

**Comment by ronhorton** [comment]
**Pass** Verified in `2.6-head Commit ID eb03517`


In values for rancher-logging
Added
fluentd:
  bufferStorageVolume:
    hostPath:
      path: ''
  disablePvc: true
/opt/logging-operator/rancher-logging-root/rancher-logging-root-fluentd-buffer
was created successfully on the file system as expected
